source/retryable-writes/retryable-writes.rst

```diff
@@ -762,6 +775,9 @@ cluster (mongos just forwards the error from the shard's replica set).
 Changes
 =======
 
+2019-10-21: Change the definition of "retryable write" to be based on the
+RetryableWriteError label.
```

Comment:
You can also add a note that we're removing old behavior to determine retryable errors from parsing the errmsg. For example:

> Removed behavior for parsing errmsg to categorized retryable errors for pre-4.4 servers.